job-dsls/jobs/deploy_jobs.groovy

```diff
@@ -70,7 +70,13 @@ def final REPO_CONFIGS = [
         ],
         "optaplanner"               : [
                 ircNotificationChannels: ["#optaplanner-dev"],
-                downstreamRepos        : ["optaplanner-wb", "optaweb-employee-rostering"]
+                downstreamRepos        : ["optaplanner-wb", "optaweb-employee-rostering"],
```

Comment:
I think there is also optaweb-vehicle-rooting downstream repository

Reply:
OK  -I will change this

Reply:
@mareknovotny DONE